packages/react/src/components/FileSelect/FileSelect.tsx

```diff
@@ -1,29 +1,32 @@
 import React from 'react';
+import { isFunction } from '@aws-amplify/ui';
+
+import { useStableId } from '../../primitives/utils/useStableId';
 
 // `FileSelect` input `type` must always be set to `file`
 const INPUT_TYPE = 'file';
 
-export type SelectionType = 'FILE' | 'FOLDER';
+export type SelectType = 'FILE' | 'FOLDER';
 
 /**
  * @internal @unstable
  */
 export interface FileSelectProps {
   accept?: string;
+  id?: string;
   multiple?: boolean;
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
-  selectionType?: SelectionType;
+  selectType?: SelectType;
   testId?: string;
 }
 
 /**
  * @internal @unstable
  */
-export interface FileSelectOptions
-  extends Omit<FileSelectProps, 'onChange' | 'type'> {}
+export interface FileSelectOptions extends Omit<FileSelectProps, 'type'> {}
```

Comment:
`type` doesn't exist in `FileSelectProps `

Reply:
Good catch. Think `type` was on an earlier version of the `FileSelectProps` interface, will update

Reply:
Updated in #5930: